setup.py

```diff
@@ -13,6 +13,7 @@
         'virtual_coach_db.dbschema': './dbschema'
     },
     include_package_data=True,
+    package_data={'': ['../resources/*.json']},
```

Comment:
You intend to have more resources json files in future?

Reply:
No, I'm restricting this to just the file we are using now.